pkg/core/localize/locales/en/cmd/dedicated.en.toml

```
@@ -0,0 +1,42 @@
+[dedicated.registerCluster.cmd.shortDescription]
+one = 'Register an OpenShift cluster with Red Hat Streams for Apache Kafka'
+
+[dedicated.registerCluster.cmd.longDescription]
+one = '''
+You can use your own OpenShift cluster to provision your kafkas which will be managed by Red Hat Streams for Apache Kafka.
+This command will register your cluster with Red Hat Streams for Apache Kafka.
+'''
+
+[dedicated.registerCluster.cmd.example]
+one = '''
+# Register an OpenShift cluster with Red Hat Streams for Apache Kafka by selecting from a list of available clusters.
+rhoas cluster register-cluster
+
+# Register an OpenShift cluster with Red Hat Streams for Apache Kafka by specifying the cluster ID.
+rhoas cluster register-cluster --cluster-id 1234-5678-90ab-cdef
+'''
+
+[dedicated.registerCluster.flag.clusterId.description]
+one = 'The ID of the OpenShift cluster to register'
```

use colons for prompts?
```suggestion
one = 'The ID of the OpenShift cluster to register:'
```